format/Schema.fbs

```diff
@@ -37,14 +37,20 @@ enum MetadataVersion:short {
   /// >= 0.8.0 (December 2017). Non-backwards compatible with V3.
   V4,
 
-  /// >= 1.0.0 (July 2020. Backwards compatible with V4 (V5 readers can read V4
+  /// 1.0.0 -> 6.0.0 (July 2020). Backwards compatible with V4 (V5 readers can read V4
   /// metadata and IPC messages). Implementations are recommended to provide a
   /// V4 compatibility mode with V5 format changes disabled.
   ///
   /// Incompatible changes between V4 and V5:
   /// - Union buffer layout has changed. In V5, Unions don't have a validity
   ///   bitmap buffer.
   V5,
+
+  /// >= 7.0.0 (Jan 2022). Backwards compatible with V4 and V5.
+  ///
+  /// Adds ColumnBag to wire format. It has looser restrictions than RecordBatch but is
+  /// otherwise similar in intent.
+  V6
```

Comment:
It doesn't look like we incremented the metadata version when we added Tensor - is this necessary? (I'm not so familiar with Flatbuffers.) I suppose if we fold the binary metadata change into this we would need this.